src/main/src/main/java/org/geoserver/security/RESTfulDefinitionSource.java

```diff
@@ -29,10 +29,11 @@ public class RESTfulDefinitionSource implements FilterInvocationSecurityMetadata
 
     private static Logger log = Logging.getLogger(RESTfulDefinitionSource.class);
 
-    private static final String[] validMethodNames = {"GET", "PUT", "DELETE", "POST"};
+    private static final String[] validMethodNames = {"GET", "PUT", "DELETE", "POST", "HEAD"};
```

Comment:
I was wondering about this change. Would you have gotten a 401 without it?
See also https://discourse.osgeo.org/t/http-options-request-fails-after-upgrade-to-2-26-1/111599/1